packages/web/src/app/components/Button.tsx

```diff
@@ -6,7 +6,7 @@ const Button = React.forwardRef(function Button(
 ) {
   return (
     <>
-      <button ref={ref} {...props} className={`button ${props.className}`} />
+      <button ref={ref} type="button" {...props} className={`button ${props.className}`} />
```

Comment:
FYI, I generally ~always try to specify `type` for `button` otherwise you can get into a bad place where a button component gets put into a form and turns into a submit button when it was meant to be a button button.